docs/data/sdks/typescript-react-native/migration.md

```diff
@@ -3,6 +3,329 @@ title: React Native SDK Migration Guide
+
+    ```json
+    {
+      "dependencies": {
+        "@amplitude/react-native": "^2"
+      }
+    }
+    ```
+
+Follow [the installation section](../#installation) to add new React Native SDK dependencies.
+
+## Instrumentation
+
+New React Native SDK offers an API to instrument events. To migrate to it, you need to update a few calls. The following sections detail which calls have changed.
+
+### Initialization
+
+`getInstance()` in the maintenance SDK only accepts API key and has been removed. To initialize the SDK, call `init()`, with the user ID and configuration parameters.
+
+=== "@amplitude/react-native"
+
+    ```typescript
+    import { Amplitude } from '@amplitude/react-native';
+
+    Amplitude.getInstance().init(API_KEY)
+    ```
+
+=== "@amplitude/analytics-react-native"
+
+    ```typescript
+    import { init } from '@amplitude/analytics-react-native';
+
+    init(API_KEY, OPTIONAL_USER_ID, config);
+    ```
+
+### Configuration
+
+However, the new React Native SDK instance includes a configuration object, allowing us to set configurations directly upon initialization via the instance.
+
+|@amplitude/react-native|@amplitude/analytics-react-native|
+|-|-|
+| `enableCoppaControl()` | NOT SUPPORTED |
+| `disableCoppaControl()` | NOT SUPPORTED |
+| `setAdvertisingIdForDeviceId()` | No configuration to set ADID as device ID. But ADID is still tracked by default as `config.trackingOptions.adid` defaults to `true`. To learn more about how device ID is initialized [here](./#device-id).  |
+| `setAppSetIdForDeviceId()` | NOT SUPPORTED |
+| `setOptOut()` | both `setOptOut()` and `config.optOut` are supported |
+| `trackingSessionEvents()` | `config.trackingSessionEvents` |
+| `setUseDynamicConfig()` | NOT SUPPORTED |
+| `setMinTimeBetweenSessionsMillis()`| similar to `config.sessionTimeout`, but they are not completely the same. Learn more about [session management](./#session-management)|
+| `setServerZone()` | `config.serverZone` |
+| `setServerUrl()` | `config.serverUrl` |
+| `setEventUploadMaxBatchSize()` | `config.flushQueueSize` |
+| `setEventUploadPeriodMillis()` | `config.flushIntervalMillis` |
+| `setEventUploadThreshold()` | `config.flushQueueSize` |
+| `enableLogging()`| Logging is enabled and cannot be turned off. However, you can set `config.logLevel` and customize `config.loggerProvider` |
+| `setLogLevel()`| `config.logLevel` |
+| `addLogCallback()` | It's not fully supported but you can customize a logger by setting `config.loggerProvider`. |
+
+### Tracking events
+
+#### `logEvent()`
+
+The `logEvent()` API maps to `track()`.
+
+=== "@amplitude/react-native"
+
+    ```typescript
+    import { Amplitude } from '@amplitude/react-native';
+    
+    Amplitude.getInstance().logEvent('Button Clicked', {buttonColor: 'primary'});
+    ```
+
+=== "@amplitude/analytics-react-native"
+
+    ```typescript
+    import { track } from '@amplitude/analytics-react-native';
+
+    track('Button Clicked', {buttonColor: 'primary'});
```

Comment:
nit: we should be consistent in the spacing here compared to `import { track }`.
```suggestion
    track('Button Clicked', { buttonColor: 'primary' });
```